ui/package.json

```diff
@@ -89,6 +89,7 @@
     "highcharts-react-official": "^2.0.0",
     "immutable": "^4.0.0-rc.9",
     "lodash": "^4.17.11",
+    "lolex": "^3.0.0",
```

Comment:
If this is only used in tests, it can go into devDependencies instead, I believe.